src/entities/protocols/seaport.ts

```diff
@@ -183,21 +216,21 @@ export class SeaportTrade extends NFTTrade<SeaportData> {
     return considerationFulfillments
   }
 
-  private getAdvancedOrderParams(data: Order): { advancedOrder: AdvancedOrder; value: BigNumber } {
+  private getAdvancedOrderParams(data: Order): { advancedOrder: AdvancedOrder } {
     const advancedOrder = {
       parameters: data.parameters,
       numerator: BigNumber.from('1'),
       denominator: BigNumber.from('1'),
       signature: data.signature,
       extraData: '0x00',
     }
-    const value = this.calculateValue(data.parameters.consideration)
-    return { advancedOrder, value }
+    return { advancedOrder }
   }
 
-  private calculateValue(considerations: ConsiderationItem[]): BigNumber {
+  private calculateValue(considerations: ConsiderationItem[], token: string): BigNumber {
     return considerations.reduce(
-      (amt: BigNumber, consideration: ConsiderationItem) => amt.add(consideration.startAmount),
+      (amt: BigNumber, consideration: ConsiderationItem) =>
+        consideration.token == token ? amt.add(consideration.startAmount) : amt,
       BigNumber.from(0)
```

Comment:
Is it possible for an order to have consideration items in different token denominations? I dont think so but good to make sure

Reply:
while i dont think that 1 consideration can, we actually sometimes submit multiple considerations in the same seaport command. So that means we could well have 2 considerations with different input currencies. Have updated to deal with this